packages/vue-client/src/assets/main.css

```diff
@@ -33,3 +33,8 @@ a,
     padding: 0 2rem;
   }
 }
+
+.column {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
Somehow I don't like the name of this class. A column can also appear in a grid layout or table layout and those will probably appear in the `main.css` too. We can leave it like this for now, but maybe I should look up some good practices for naming CSS classes.

Reply:
renamed to config-form-column